posts/cassandra_models_benchmarking.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Benchmarking Cassandra models
+author: carlos
+date: "2015-05-14"
+description: An example of how we benchmark our Cassandra models before deploying them
+tags: ["Cassandra", "Performance", "DataModelling", "Benchmarking"]
+categories: ["development", "profiling", "data modelling"]
+---
+
+During the last couple of weeks we've been very focused on deeply understanding Cassandra's
+write and read paths to build our models the best possible way.
+
+After so much investigation I can summarize which are the steps we follow to assess our
+model's performance.
+
+### 1. Have good practices and anti-patterns always in mind.
+
+Cassandra is known to be a very resilient and highly performant platform, but it will
```

Comment:
, but only so long as you follow the rules and work *with* the underlying data model.